optimize/upgrade/src/test/java/io/camunda/optimize/upgrade/migrate313to86/AbstractUpgrade86IT.java.out

```diff
@@ -8,33 +8,46 @@
 // TODO recreate C8 IT equivalent of this with #13337
 // package io.camunda.optimize.upgrade.migrate313to86;
 // 
-// import io.camunda.optimize.upgrade.AbstractUpgradeIT;
-// import io.camunda.optimize.upgrade.migrate313to86.indices.OnboardingStateIndexV2;
-// import io.camunda.optimize.upgrade.migrate313to86.indices.SettingsIndexV2;
-// import io.camunda.optimize.upgrade.plan.UpgradePlan;
-// import io.camunda.optimize.upgrade.plan.UpgradePlanRegistry;
-// import java.util.List;
-// import org.junit.jupiter.api.BeforeEach;
-// 
-// public class AbstractUpgrade86IT extends AbstractUpgradeIT {
-// 
-//   protected static final String FROM_VERSION = "3.13.0";
-//   protected static final String TO_VERSION = "8.6.0";
-// 
-//   protected final OnboardingStateIndexV2 ONBOARDING_STATE_INDEX = new OnboardingStateIndexV2();
-//   protected final SettingsIndexV2 SETTINGS_INDEX = new SettingsIndexV2();
-// 
-//   @BeforeEach
-//   protected void setUp() throws Exception {
-//     super.setUp();
-//     initSchema(List.of(ONBOARDING_STATE_INDEX, SETTINGS_INDEX));
-//     setMetadataVersion(FROM_VERSION);
-//   }
-// 
-//   protected void performUpgrade() {
-//     final List<UpgradePlan> upgradePlans =
-//         new UpgradePlanRegistry(upgradeDependencies)
-//             .getSequentialUpgradePlansToTargetVersion(TO_VERSION);
-//     upgradePlans.forEach(plan -> upgradeProcedure.performUpgrade(plan));
-//   }
-// }
+//import java.util.List;
```

Comment:
What's the point of this file and the file below? Why is everything commented out?

Reply:
For Optimize8, we have commented out all IT that currently only runs in C7 mode but should eventually be migrated to also run in C8 mode. The idea is to basically have all these test files we can migrate one by one, removing the C7 one and adding the C8. We've removed all IT that is C7 exclusive and does not need migrating.